### PR TITLE
remove project.json string

### LINKED ISSUE #2722: convert aspnetcore/fundamentals/localization/ from project.json to csproj
sample/Resources/Views/Book/Index.resx 
Also other language .resx files
sample/Views/

### PR DESCRIPTION
**Pull request by Rick-Anderson**
fixes #2722

### CODE REVIEW

**Comment by Rick-Anderson** [comment]
@tdykstra  please review

**Comment by hishamco** [comment]
@Rick-Anderson looks good

File: aspnetcore/fundamentals/localization/sample/Resources/Views/Book/Index.resx [hunk 1]
```diff
@@ -198,8 +198,8 @@
   <data name="Run &amp; Deploy" xml:space="preserve">
     <value>Déployer &amp; Exécuter</value>
   </data>
-  <data name="Run commands in your project.json" xml:space="preserve">
-    <value>Exécuter des commandes dans votre project.json</value>
+  <data name="Run commands in your bash" xml:space="preserve">
```
Comment (tdykstra) [inline aspnetcore/fundamentals/localization/sample/Resources/Views/Book/Index.resx]: What does "Run commands in your project.json" doesn't make sense to me and "bash" doesn't sound like a direct replacement for "project.json".  Can you explain what's going on here?

Comment (Rick-Anderson) [inline aspnetcore/fundamentals/localization/sample/Resources/Views/Book/Index.resx]: Lots of the sentences  don't make sense. I could probably just remove all those lines. that referred to project.json. This is just to demo how to localize strings so the actual string doesn't really matter. It doesn't need to make sense. There is a big array of strings to localize and the sample only uses a few of them.

**Pull request merged**